src/qibolab/instruments/qm.py

```diff
@@ -709,7 +712,12 @@ def sweep_frequency(self, sweepers, qubits, qmsequence, relaxation_time):
             else:
                 raise_error(NotImplementedError, f"Cannot sweep frequency of pulse of type {pulse.type}.")
             # convert to IF frequency for readout and drive pulses
-            freqs0.append(declare(int, value=int(pulse.frequency - lo_frequency)))
+            f0 = int(pulse.frequency - lo_frequency)
+            freqs0.append(declare(int, value=f0))
+            # check if sweep is within the supported bandwidth [-400, 400] MHz
+            max_freq = max(abs(min(sweeper.values) + f0), abs(max(sweeper.values) + f0))
```

Comment:
I'd expect:
```py
abs(min(sweeper.values) + f0) < abs(max(sweeper.values) + f0)
```
why this is not the case?

Reply:
Is it actually possible that:
```py
max(sweeper.values) + f0 < 0
```
?

Reply:
> I'd expect:
> 
> ```python
> abs(min(sweeper.values) + f0) < abs(max(sweeper.values) + f0)
> ```
> 
> why this is not the case?

`f0` is an IF so it can be negative, so for example if:
* `f0 = -200`
* `min(sweeper.values) = -100`
* `max(sweeper.values) = 100`

the above inequality is violated.